src/pages/download/index.tsx

```diff
@@ -30,7 +30,10 @@ const DownloadPage = ({ data: { nodeReleases } }: Props): JSX.Element => {
     release => release.status !== 'End-of-life'
   );
 
-  const lts = filteredReleases.find(release => release.isLts);
+  const lts = filteredReleases
```

Comment:
This change shouldn't be done here, and it will only "affect" the download page. As I mentioned the change should be done on `gatsby-node` to actually fix the root cause.

Reply:
Ah okay, I'll get it done.